ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml

```diff
@@ -21,15 +21,15 @@ tests:
   postsubmit: true
   steps:
     env:
-      DESTINATION_BRANCH: release-2.15
+      DESTINATION_BRANCH: release-2.16
     workflow: ocm-ci-fastforward
 - as: deploy-and-run-e2e
   skip_if_only_changed: ^docs/|\.(md|adoc)$|^(README|LICENSE|OWNERS)$|^tests/(api|cypress)/.*$|^Jenkins.*$
   steps:
     dependencies:
       COMPONENT_IMAGE_REF: search-e2e
     env:
-      ACM_RELEASE_VERSION: release-2.15
+      ACM_RELEASE_VERSION: release-2.16
```

Comment:
Keep 2.15 here or tests will fail because there isn't a 2.16 build promoted yet. We'll change it after 2.16 is available.

Reply:
Same for the 2 instances below.